src/golang/module.go

```diff
@@ -27,10 +27,10 @@ import (
 )
 
 /*
-#cgo CFLAGS: -I/usr/include/bcc/compat
-#cgo LDFLAGS: -lbcc
-#include <bcc/bcc_common.h>
-#include <bcc/libbpf.h>
+#cgo CFLAGS: -I/${SRCDIR}/../cc
+#cgo LDFLAGS: -L/${SRCDIR}/../../build/src/cc -lbcc
+#include <bcc_common.h>
+#include <libbpf.h>
```

Comment:
This change seems wrong to me.

`go get` will not download these headers, so you are expecting users to use this module locally.

Reply:
I was looking how it is done elsewhere in libseccomp and libsecomp-golang:
https://github.com/seccomp/libseccomp-golang/blob/9006a281d6713ba604f36f9b5caeac23a16bbd55/seccomp_internal.go#L16-L31

They use `#cgo pkg-config: libseccomp` so the paths don't have to be hardcoded. Then, they have instructions in comment to explain how to use `PKG_CONFIG_PATH` and `LD_PRELOAD` to use a local library. So it is possible to support both system-installed libbcc and local module.

They have a compile-time version check (see `#error This package requires libseccomp >= v2.3.1`). Maybe we can do the same with LIBBCC_VERSION.

They also have run-time version check with `seccomp_api_get()` but I don't think we need that in bcc at this point.